code-challenges.test.js

```diff
@@ -14,14 +14,48 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
+
+
 const people = [
   { name: "ford prefect", occupation: "a hitchhiker" },
   { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
   { name: "arthur dent", occupation: "a radio employee" }
 ]
+
+
+describe("sentenceArr", () => {
```

Comment:
Excellent work on this problem!!  I would recommend placing your test variables within the test so they are localized to the scope of the test.